runtime/sema/capability_controller.cdc

```diff
@@ -0,0 +1,35 @@
+pub struct CapabilityController {
```

Comment:
Please add some checker tests for this new type

Reply:
hmm, added some rudimentary ones asserting the type exists in scope and conforms to the spec. will add some proper tests for the account capabilities methods when this PR is in, since those involve non-trivial generics.